README.md

```diff
@@ -4,7 +4,6 @@
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://juliatesting.github.io/Aqua.jl/dev)
 [![GitHub Actions](https://github.com/JuliaTesting/Aqua.jl/workflows/Run%20tests/badge.svg)](https://github.com/JuliaTesting/Aqua.jl/actions?query=workflow%3ARun+tests)
 [![Codecov](https://codecov.io/gh/JuliaTesting/Aqua.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaTesting/Aqua.jl)
-[![GitHub commits since tagged version](https://img.shields.io/github/commits-since/JuliaTesting/Aqua.jl/v0.6.2.svg)](https://github.com/JuliaTesting/Aqua.jl)
```

Comment:
Alternatively, you could use
```suggestion
[![GitHub commits since tagged version](https://img.shields.io/github/commits-since/JuliaTesting/Aqua.jl/latest.svg)](https://github.com/JuliaTesting/Aqua.jl)
```

Reply:
Does that really work without updating a `latest` tag? Is there any documentation for this, i.e., how this badge is supposed to work?

Reply:
I've never used it, just came across it when browsing https://github.com/badges/shields. It was added in https://github.com/badges/shields/pull/1115 and as far as I understand it's supposed to always compare with the latest release (no latest tags required).

Reply:
It seems there are even more options (regarding e.g. order etc.): https://shields.io/category/activity